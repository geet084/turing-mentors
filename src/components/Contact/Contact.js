import React, { Component } from 'react';

export class Contact extends Component {
  render() {
    return (
      <div>
        <h2>Contact page</h2>
        <p>Turing contact info...</p>
      </div>
    )
  }
}

export default Contact;