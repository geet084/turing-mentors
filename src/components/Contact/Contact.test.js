import React from 'react';
import { shallow } from 'enzyme';
import { Contact } from './Contact';

describe('Contact', () => {
  let wrapper;
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<Contact />);
    expect(wrapper).toMatchSnapshot();
  });
});