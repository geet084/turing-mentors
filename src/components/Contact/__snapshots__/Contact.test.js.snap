// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should have match the correct snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Turing School of Software and Design
        </h2>,
        <p>
          1331 17th Street
        </p>,
        <p>
          Suite LL100
        </p>,
        <p>
          Denver, CO 80202
        </p>,
        <p>
          Phone: 303.731.3117
        </p>,
        <p>
          Fax: 303.374.7360
        </p>,
      ],
      "className": "link-content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Turing School of Software and Design",
        },
        "ref": null,
        "rendered": "Turing School of Software and Design",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1331 17th Street",
        },
        "ref": null,
        "rendered": "1331 17th Street",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Suite LL100",
        },
        "ref": null,
        "rendered": "Suite LL100",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Denver, CO 80202",
        },
        "ref": null,
        "rendered": "Denver, CO 80202",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Phone: 303.731.3117",
        },
        "ref": null,
        "rendered": "Phone: 303.731.3117",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Fax: 303.374.7360",
        },
        "ref": null,
        "rendered": "Fax: 303.374.7360",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Turing School of Software and Design
          </h2>,
          <p>
            1331 17th Street
          </p>,
          <p>
            Suite LL100
          </p>,
          <p>
            Denver, CO 80202
          </p>,
          <p>
            Phone: 303.731.3117
          </p>,
          <p>
            Fax: 303.374.7360
          </p>,
        ],
        "className": "link-content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Turing School of Software and Design",
          },
          "ref": null,
          "rendered": "Turing School of Software and Design",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1331 17th Street",
          },
          "ref": null,
          "rendered": "1331 17th Street",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Suite LL100",
          },
          "ref": null,
          "rendered": "Suite LL100",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Denver, CO 80202",
          },
          "ref": null,
          "rendered": "Denver, CO 80202",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Phone: 303.731.3117",
          },
          "ref": null,
          "rendered": "Phone: 303.731.3117",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Fax: 303.374.7360",
          },
          "ref": null,
          "rendered": "Fax: 303.374.7360",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
